detection_rules/schema.py

```diff
@@ -81,6 +82,23 @@ class Filters(jsl.Document):
     query = jsl.DocumentField(FilterQuery)
 
 
+class RiskScoreMapping(jsl.Document):
+    """Risk score mapping"""
+
+    field = jsl.StringField(required=True)
+    operator = jsl.StringField(required=True, enum=OPERATORS)
+    value = jsl.StringField(required=True, default="")
```

Comment:
Initially we're only supporting the copying a single field's value to `event.risk_score` for `RiskScoreMapping`, however the intent here is to allow for more complex mapping in future releases. We'll probably need a `risk_score` field here as well (like `severity` below) for mapping to specific values, e.g. `some.custom.field 'equals' some_value` maps to the provided `risk_score`. 

To be honest, it would be nice to have an expression engine where the user could have access to all source event fields and then calculate the desired `risk_score`. Would appreciate any thoughts/ideas here 🙂

Reply:
I don't thinks this was intended?
```suggestion
    field = jsl.StringField(required=True)
    operator = jsl.StringField(required=True, enum=OPERATORS)
    value = jsl.StringField(required=True)
```

Reply:
Looks like I just conflated `required`/`default` -- I think we'll want `default` here?